app/controllers/api/v1/api_keys_controller.rb

```diff
@@ -48,6 +48,9 @@ def update
 
   def check_mfa(user)
     if user&.mfa_gem_signin_authorized?(otp)
+      return render_mfa_forbidden(mfa_setup_required_error) if user.mfa_required_not_yet_enabled?
```

Comment:
`render_mfa_forbidden` seems to just render the text given and it isn't mfa specific. Looking at the [current patterns with base controller errors](https://github.com/rubygems/rubygems.org/blob/298995cc5a1f6f24603b8ca21fd4b649cd167ea8/app/controllers/api/base_controller.rb#L110-L124), I think it makes sense to just incorporate render into  `mfa_setup_required_error` and `mfa_strong_level_required_error` methods.

Reply:
@jenshenny I tried that initially but it failed code climate because having the render in both places is too repetitive, it wanted me to DRY it

Reply:
Ack, code climate! Do you recall what the exact violation is? Would something like this work? I don't appreciate having to call `render_mfa_forbidden` with the error if in all cases we would render the error anyways.
```
def render_mfa_setup_required_error
error = "..."
render_forbidden(error)
end

def render_mfa_strong_level_required_error
error="..."
render_forbidden(error)
end

def render_forbidden(error)
respond_to...
end
```